### PR TITLE
COMMERCE-9253 Clear selections in Commerce Search Facets

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-9253

sent from: #3564 

@CrescenzoRegaLF

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 1]
```diff
@@ -240,6 +240,114 @@ definition {
...
+		task ("And the buyer navigates to the Minium Catalog") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("When clicks some search fields in Option Facet") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:272]:
```suggestion
		task ("When the user selects some search fields in Option Facet") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 2]
```diff
@@ -240,6 +240,114 @@ definition {
...
+					fieldName = "Option Facet",
+					fieldValue = ${searchFieldValue});
+			}
+
+			AssertElementPresent(
+				key_productCount = 49,
+				locator1 = "CommerceFrontStore#FS_PRODUCT_COUNT");
+		}
+
+		task ("When clicks some search fields in Specification Facet") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:298]:
```suggestion
		task ("When the user selects some search fields in Specification Facet") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase [hunk 3]
```diff
@@ -240,6 +240,114 @@ definition {
...
+					fieldName = "Specification Facet",
+					fieldValue = ${searchFieldValue});
+			}
+
+			AssertElementPresent(
+				key_productCount = 49,
+				locator1 = "CommerceFrontStore#FS_PRODUCT_COUNT");
+		}
+
+		task ("When clicks some search fields in Price Range Facet") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceFacets.testcase:324]:
```suggestion
		task ("When the user selects some search fields in Price Range Facet") {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:forward

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**